src/components/LogRow/ResmokeRow/index.tsx

```diff
@@ -1,19 +1,20 @@
 import { forwardRef } from "react";
+import CollapsedRow from "components/LogRow//CollapsedRow";
 import BaseRow from "components/LogRow/BaseRow";
 import { BaseRowProps } from "../types";
 
 const ResmokeRow = forwardRef<any, BaseRowProps>((rowProps, ref) => {
   const { data, listRowProps } = rowProps;
   const { getLine, wrap, processedLines } = data;
   const { index } = listRowProps;
+
   const line = processedLines[index];
 
-  // TODO: EVG-17535
-  // This should be replaced with a collapsible component
-  const lineIndex = Array.isArray(line) ? line[0] : line;
-  return (
-    <BaseRow ref={ref} wrap={wrap} {...listRowProps}>
-      <span data-cy="resmoke-row">{getLine(lineIndex)}</span>
+  return Array.isArray(line) ? (
+    <CollapsedRow ref={ref} {...listRowProps} numCollapsed={line.length} />
+  ) : (
+    <BaseRow ref={ref} wrap={wrap} {...listRowProps} lineNumber={line}>
+      <span data-cy="resmoke-row">{getLine(line)}</span>
     </BaseRow>
```

Comment:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/4605522/190246446-a0880b86-cfbc-47ae-9627-a0470dcdf83a.png">
I still see the weird styling for resmoke rows.

Reply:
I think it makes sense to align the lines so the buttons line up